OPTIONS.md

```diff
@@ -858,7 +858,7 @@ Commands:
   configtest|t [config-path]                                                         Validate a webpack configuration.
   help|h [command] [option]                                                          Display help for commands and options.
   info|i [options]                                                                   Outputs information about your system.
-  init|c [scaffold...] [options]                                                     Initialize a new webpack configuration.
+  init|c [scaffold...] [options]                                                     Initialize a new webpack project.
```

Comment:
I think we should change signature, because was changed

Reply:
fixed